### ISSUE TITLE
Deriving rate of VTE or rate of VTE-related death

### ISSUE THREAD

**Issue by matthewberesford92** [opensafely/vte-diagnoses-and-deaths#6]
@Jongmassey 

@joshuamutio  and I discussed today re deriving monthly rates for VTE and VTE-related death over the period of the study. For the denominator we would need a count of the total population within TPP for that month. This clearly requires info not captured in our defined study population. 

How is normally managed as deriving population rates must be commonplace?

Thanks
Matt

**Comment by Jongmassey** [comment]
Yes this commonly done, usually with our `Measures` framework. I've not used this framework with ehrql yet but will investigate for you.

If the denominator population is different from the study population then effectively you will end up defining a new dataset for the denominator population, creating a measure to do the population count by month, then creating a separate measure for the numerator based on the primary study population, then combining these outputs downstream to get a monthly rate
